docs/methoddocs/archive.md

```diff
@@ -1,7 +1,7 @@
 # Archive
 
 ```eval_rst
-.. autoclass:: py42.modules.archive.ArchiveModule
```

Comment:
What about storage archive servife?

Reply:
it is not currently directly exposed. The plan is to eventually make it easier to create the individual services "standalone" so that people could call those without creating the whole sdk, but right now nothing has changed in terms of what users are expected to call directly.